website/i18n.js

```diff
@@ -19,6 +19,9 @@ const localeConfigs = {
   'es-ES': {
     label: 'Español',
   },
+  'fr': {
```

Comment:
You also need to add it to the locales array above

BTW it's now not-required to provide a local config, as Docusaurus provide default localized labels for each locale automatically

Reply:
Oops ! i fix it !